File: JavaScript solutions/122. Best Time to Buy and Sell Stock II.js
/**
 * @param {number[]} prices
 * @return {number}
 */
var maxProfit = function(prices) {
    var sum=0;
    for(var i=1;i<prices.length;i++)
    sum+=Math.max(prices[i]-prices[i-1],0);
    return sum;
};

File: C++ solutions/122. Best Time to Buy and Sell Stock II.cpp
#include<bits/stdc++.h>
using namespace std;

// solution
class Solution {
public:
    int maxProfit(vector<int>& prices) {
        int sum=0;
        for(int i=1;i<prices.size();i++)
            sum+=max(prices[i]-prices[i-1],0);
        return sum;
    }
};